code/modules/mechs/mech.dm

```diff
@@ -194,5 +194,10 @@
 /mob/living/exosuit/proc/return_temperature()
 	return bodytemperature
 
+/mob/living/exosuit/proc/get_pilots()
+	return pilots?.len ? pilots : null
```

Comment:
Why is that a thing? Isn't it easier to move the code entirely to `get_mob`?

Reply:
Cuz it used not only in get_mob, it used in simple animals SSatackable too

Reply:
I don't see a reason why simple animals would use a snowflake proc and not some universal shit like `length(exosuit.pilots)`.